internal/pkg/manifest/pipeline.go

```diff
@@ -97,7 +97,8 @@ type Source struct {
 
 // PipelineStage represents a stage in the pipeline manifest
 type PipelineStage struct {
-	Name string `yaml:"name`
+	Name         string   `yaml:"name"`
+	TestCommands []string `yaml:"test_commands"`
```

Comment:
Can we include:
```suggestion
	TestCommands []string `yaml:"test_commands,omitempty"`
```
So that when the file is marshaled the field doesn't show up if the slice is nil.

Reply:
Yup, thanks!